model/src/main/kotlin/OrtResult.kt

```diff
@@ -289,6 +289,11 @@ data class OrtResult(
         return vendorPackages
     }
 
+    @Suppress("UNUSED") // This is intended to be mostly used via scripting.
+    fun getUncuratedPackageById(id: Identifier): Package? =
+        analyzer?.result?.packages?.find { it.pkg.id == id }?.toUncuratedPackage() ?:
+        analyzer?.result?.projects?.find { it.id == id }?.toPackage()
```

Comment:
This line should have more indentation. Personally, I would prefer to put the `?:` on the beginning of this line, I think we format like this in most other places.